test/Auto/build.sh

```diff
@@ -0,0 +1,64 @@
+echo "Clone repo start"
+sudo git clone https://github.com/F5Networks/f5-adcaas-openstack.git
+
+echo "Apt update start"
+sudo apt-get update
+
+echo "Nodejs install start"
+sudo apt-get install nodejs -y
+
+echo "Npm install start"
+sudo apt-get install npm -y
+
+echo "Docker install start"
+sudo apt-get install apt-transport-https ca-certificates curl gnupg-agent     software-properties-common -y
+sudo apt install docker.io -y
+
+echo "Typescript install start"
+sudo npm install -g typescript -y
+
+echo "Docker compose install start"
+# sudo curl -L "https://github.com/docker/compose/releases/download/1.24.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+# sudo chmod +x /usr/local/bin/docker-compose
+
+sudo npm install --save-dev eslint-config-prettier@^5.0.0
+# sudo npm install --save-dev eslint@^1.1.2
+
+sudo systemctl start docker
+
+echo "App build and install start"
+cd f5-adcaas-openstack/app/waf
+
+echo "Npm install start"
+sudo npm install
+
+echo "Npm build start"
+sudo npm run build
+
+echo "Portal install start"
+sudo docker build . -t f5devcentral/f5-openstack-services:portal-latest
```

Comment:
Remove sudo.  You may add ubuntu user to docker group, if it can not run docker command.